### ISSUE TITLE
I built a small webapp for managing xmrig-proxy with the new API

### ISSUE THREAD

**Issue by pcca-matrix** [xmrig/xmrig-proxy#186]
Maybe you find it useful
Feel free to use it

[XMRig-Proxy-Frontend](https://github.com/pcca-matrix/XMRig-Proxy-Frontend)

Cheers,

**Comment by ORGaniZMo** [comment]
@pcca-matrix 
Do you use Access Token for authorization?

**Comment by pcca-matrix** [comment]
Yes it support access token it's mandatory to change the config on the fly but only on the server side for now (hardcoded in php script) , since i only use it with VPN, security is not a problem.

When i have some free time i'm adding an app password to communicate securely with php script ,
 it's on my todo list.

**Comment by ORGaniZMo** [comment]
> When i have some free time i'm adding an app password to communicate securely with php script ,
it's on my todo list.
I'll be waiting...

**Comment by ariadarkkkis** [comment]
Can you make this for xmr-node-proxy please?

**Comment by pcca-matrix** [comment]
@ariadarkkkis 
I do not use it , maybe later if api is similar

@ORGaniZMo 
App password and multiple proxy instance is now added

**Comment by ariadarkkkis** [comment]
@pcca-matrix I dont think xmr-node-proxy have api. Its just node-js. I did some work but because I dont have any knowledge of this language its hard a bit
